src/components/accessories/patientTherapy/PatientTherapy.tsx

```diff
@@ -75,10 +71,14 @@ const PatientTherapy: FC<TProps> = ({
   const resetFormCallback = () => {
     setShouldResetForm(false);
     setShouldUpdateTable(false);
+    dispatch(deleteTherapyReset());
     setActivityTransitionState("IDLE");
     scrollToElement(null);
   };
-
+  const onDelete = (code: number | undefined) => {
+    setDeletedObjCode(`${code ?? ""}`);
```

Comment:
See the https://github.com/informatici/openhospital-ui/pull/167#discussion_r685069816